org.eclipse.winery.model.tosca/src/main/java/org/eclipse/winery/model/tosca/constants/Namespaces.java

```diff
@@ -11,18 +11,46 @@
  *******************************************************************************/
 package org.eclipse.winery.model.tosca.constants;
 
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
 /**
  * Defines namespace constants not available in Java7's XMLConstants
  */
 public class Namespaces {
 
-	public static final String TOSCA_NAMESPACE = "http://docs.oasis-open.org/tosca/ns/2011/12";
-	public static final String TOSCA_WINERY_EXTENSIONS_NAMESPACE = "http://www.opentosca.org/winery/extensions/tosca/2013/02/12";
+    public static final String TOSCA_NAMESPACE = "http://docs.oasis-open.org/tosca/ns/2011/12";
+    public static final String TOSCA_WINERY_EXTENSIONS_NAMESPACE = "http://www.opentosca.org/winery/extensions/tosca/2013/02/12";
+
+    // XML Schema namespace is defined at Java7's XMLConstants.W3C_XML_SCHEMA_NS_URI
+
+    public static final String URI_BPMN20_MODEL = "http://www.omg.org/spec/BPMN/20100524/MODEL";
+    public static final String URI_BPMN4TOSCA_20 = "http://www.opentosca.org/bpmn4tosca";
+    public static final String URI_BPEL20_ABSTRACT = "http://docs.oasis-open.org/wsbpel/2.0/process/abstract";
+    public static final String URI_BPEL20_EXECUTABLE = "http://docs.oasis-open.org/wsbpel/2.0/process/executable";
+
+    /**
+     * Namespaces moved from ConfigurationBasedNamespaceManager.java
+     **/
+
+    public static final String W3C_SCHEMA = "http://www.w3.org/2001/XMLSchema";
+    public static final String W3C_NAMESPACE = "http://www.w3.org/2001/namespace";
+    public static final String EXAMPLE_NAMESPACE = "http://www.example.org";
+    public static final String URI_OPENTOSCA_NODETYPE = "http://www.opentosca.org/nodetypes";
+
+    // added from TDocumentation.java
```

Comment:
No need to comment that.

Reply:
Order these alphabetically. W3C should stand together with the other W3C namespace.